Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "helius"
-version = "0.2.1"
+version = "2.0.0"
```

Comment:
To reflect that this uses 2.x of solana-sdk

Reply:
I don't agree with the huge version bump to 2.0.0 — I understand the rationale to be on par with other Solana SDK versions. Still, it doesn't make sense within this context as the semver is specific to the Helius SDK and this is technically just a dependency update. Can you please bump it to 0.2.2?